Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -352,29 +352,30 @@ def condition(self, fragment, stepList, key, hasHarvest):
     offset = 0.9,
 )
 
-# MLPF workflows
-class UpgradeWorkflow_mlpf(UpgradeWorkflow):
+
+# Track DNN workflows
+class UpgradeWorkflow_trackdnn(UpgradeWorkflow):
     def setup_(self, step, stepName, stepDict, k, properties):
-        if 'Reco' in step:
-            stepDict[stepName][k] = merge([self.step3, stepDict[step][k]])
+        stepDict[stepName][k] = merge([{'--procModifiers': 'trackdnn'}, stepDict[step][k]])
+        custNew = "RecoTracker/FinalTrackSelectors/customiseForDNN.customiseForDNN"
+        if '--customise' in stepDict[stepName][k].keys():
+            stepDict[stepName][k]['--customise'] += ","+custNew
+        else:
+            stepDict[stepName][k]['--customise'] = custNew
+
     def condition(self, fragment, stepList, key, hasHarvest):
         return fragment=="TTbar_14TeV" and '2021' in key
-
-upgradeWFs['mlpf'] = UpgradeWorkflow_mlpf(
```

Comment:
why are you removing this existing upgrade workflow?

Reply:
I believe this is unintentional, maybe a result of the rebase. Minxi, can you please check and fix this?

Reply:
I checked it. This is accidently deleted when I solve the conflict during the rebase. I will fix it soon.